pytest_timeouts.py

```diff
@@ -172,7 +171,9 @@ def fetch_marker_timeout(item, name):
         def get_fixture_scope(item):
             return item._fixtureinfo.name2fixturedefs[
                 item._fixtureinfo.names_closure[0]][0].scope
-        markers = TimeoutsPlugin.get_markers(item, name)
+
+        get_markers = getattr(item, TimeoutsPlugin.get_markers_func_name)
```

Comment:
What's the benefit from invoke getattr instead of explicit invoke function?

Reply:
1) less call stack
2) less code
But invoke works faster. Changed